src/Compilers/CSharp/Test/Symbol/Symbols/Source/NullablePublicAPITests.cs

```diff
@@ -3815,5 +3815,40 @@ void verify(DefaultExpressionSyntax expr, PublicNullableAnnotation expectedAnnot
                 Assert.Equal(expectedState, info.FlowState);
             }
         }
+
+        [Fact]
+        public void GetTypeInfoOnNullableType()
```

Comment:
>GetTypeInfoOnNullableType [](start = 20, length = 25)

Please add tests for type references in declarations.

Reply:
Added.

---
In reply to: [366368793](https://github.com/dotnet/roslyn/pull/40950#discussion_r366368793) [](ancestors = 366368793)

Reply:
> Added.

Sorry for not making it clear that I meant in declarations outside of executable code. I.e. in member declarations, etc. 

---
In reply to: [366588587](https://github.com/dotnet/roslyn/pull/40950#discussion_r366588587) [](ancestors = 366588587,366368793)

Reply:
I think type parameters for constructed types and methods in an executable code also are not backed by BoundTypeExpression nodes. Consider testing them too.

---
In reply to: [366616630](https://github.com/dotnet/roslyn/pull/40950#discussion_r366616630) [](ancestors = 366616630,366588587,366368793)